react-frontend/src/css/pagetop.css

```diff
@@ -36,6 +26,8 @@
 }
 
 .plainPageText {
+    display:block;
```

Comment:
Is this class name being used anywhere now? (I could not find.) If not, delete?

Reply:
good catch, I have deleted it.